documentation/how-to/tracking.en.md

```diff
@@ -5,89 +5,92 @@ tx_slug: documentation_how-to_tracking
 
 # Tracking
 
-QField allows you to create points, lines, and polygons from tracking your position.
+:material-tablet: Fieldwork
 
-## Activate tracking
+QField allows you to track your position through the creation of points, lines or polygons while browsing the map, working on other features and layers, or in the background while having the device in your pocket.
+To enable tracking, you need to ensure that your positioning is activated.
 
-:material-tablet: Fieldwork
+## Set-Up Tracking Session
 
-To activate tracking, you must first make sure the positioning is active. Then,
-open the side dashboard, long-press on a legend layer within which you want to
-save your tracks to, and select the *Setup tracking* button to configure
-the tracking session.
+1. Open the main dashboard legend and press long on the layer which you want to use for tracking.
+2. Select **Setup tracking** and a new window will appear showing the configuration settings.
 
-!![](../assets/images/tracking-layer-properties.png)
+    !![](../assets/images/tracking-layer-properties.png,350px)
 
-Two constraints are available to limit the number of vertices recorded
-during tracking:
-- A minimum time interval
-- A minimum distance
+    You can choose between two options how the number of vertices will be recorded during the tracking session:
 
-!![](../assets/images/tracking-settings.png)
+    - The `Time requirement` records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
+    - The `Distance requirement` is more efficient, recording a point only after you move a minimum distance.
+    This keeps your track clean by ignoring movement when you are stationary.
+    To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.
 
-Once you have finished configuring your tracking session, hit the *Start tracking*
-button to begin recording. At this stage, a feature form will appear, which allows
-you to define the attributes of the feature(s) that will be created while QField
-tracks your position.
+    !![](../assets/images/tracking-settings.png,350px)
+    - The erroneous distance safeguard option can be turned on to insure sporadic bad GNSS readings are not tracked.
+    The functionality relies on providing a maximum tolerated distance from the last recorded position beyond which readings will be skipped.
+    In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.
```

Comment:
```suggestion
    To prevent the next vertex from being set too far from the most recent one (e.g., in steep terrain where elevation matters), set a maximum tolerance distance.
```

Shorter, easier to read